ledger-service/http-json-cli/src/main/scala/com/daml/http/OptionParser.scala

```diff
@@ -165,4 +170,16 @@ class OptionParser(getEnvVar: String => Option[String], supportedJdbcDriverNames
     .optional()
     .valueName(WebsocketConfig.usage)
     .text(s"Optional websocket configuration string. ${WebsocketConfig.help}")
+
+  // TODO: Refactor out as this is duplicated from the sandbox code
+  opt[String]("log-level")
+    .optional()
+    .validate(l =>
+      Either
+        .cond(KnownLogLevels.contains(l.toUpperCase), (), s"Unrecognized logging level $l")
+    )
+    .action((level, c) => c.copy(logLevel = Some(LogLevel.toLevel(level.toUpperCase))))
```

Comment:
```suggestion
  opt[LogLevel]("log-level")
    .optional()
    .action((level, c) => c.copy(logLevel = Some(level)))
```

```scala
// elsewhere
implicit val scoptLogLevel: scopt.Read[LogLevel] = scopt.Read.reads { l =>
  // KnownLogLevels check and toLevel here
}
```

Reply:
very good idea, in my initial code when I didn't copy over that code from somewhere else of the codebase I had such an implicit scopt.Read instance too.

Reply:
I will address this with the refactoring PR.